test/smoke_test.py

```diff
@@ -29,6 +27,13 @@ def smoke_test_torchvision_read_decode() -> None:
         raise RuntimeError(f"Unexpected shape of img_png: {img_png.shape}")
 
 
+def smoke_test_torchvision_decode_jpeg_cuda():
+    img_jpg_data = read_file(str(SCRIPT_DIR / "assets" / "encode_jpeg" / "grace_hopper_517x606.jpg"))
+    img_jpg = decode_jpeg(img_jpg_data, device="cuda")
+    if img_jpg.ndim != 3 or img_jpg.numel() < 100:
+        raise RuntimeError(f"Unexpected shape of img_jpg: {img_jpg.shape}")
```

Comment:
Why not check the shape exactly? Pixel values may vary slightly but the image shape should always be the same

Reply:
Happy to do so, but was just following what we had for the CPU variant:

https://github.com/pytorch/vision/blob/9bc094ea5e7cb90f1b1780508c95eaab4ffb1d74/test/smoke_test.py#L23-L29